lib/gpu/lib/src/shader.dart

```diff
@@ -0,0 +1,35 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: public_member_api_docs
+
+part of flutter_gpu;
+
+base class UniformSlot {
+  UniformSlot(this.slotId, this.shaderStage);
+
+  int slotId;
+  ShaderStage shaderStage;
```

Comment:
Both of these fields feel like they should be `final` and the `class` const

Reply:
Done.